### PR TITLE
After clicking on alert, navigate to linelist filtered for upload id

### PR DESCRIPTION
**Pull request by allysonjp715**
ezgif com-video-to-gif (1)

### CODE REVIEW

File: verification/curator-service/ui/cypress/support/commands.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ declare global {
 }
 
 export function addCase(opts: {
+    uploadId?: string;
```
Comment (axmb) [inline verification/curator-service/ui/cypress/support/commands.ts:32]: nit, but optional params should be after non-optional params (and this part might need to be merged)

File: verification/curator-service/ui/src/components/Alerts.tsx [hunk 2]
```diff
@@ -84,24 +98,43 @@ export default function Alerts(): JSX.Element {
...
-                            upload.summary.numUpdated,
+                            uploadData.upload.summary.numCreated,
+                            uploadData.upload.summary.numUpdated,
                         )}
+                        {' from '}
+                        <a
+                            href={createHref(uploadData.sourceUrl)}
+                            rel="noopener noreferrer"
+                            target="_blank"
+                            onClick={(e): void => e.stopPropagation()}
```
[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Alerts.tsx:131]: It stops the event from being passed to the next div. So without this clicking on the link would open this link in a new tab, but also trigger clicking on the alert which would navigate the page to /cases.

File: verification/curator-service/ui/cypress/support/commands.ts (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/cypress/support/commands.ts]: Done

File: verification/curator-service/ui/src/components/Alerts.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (axmb) [reply verification/curator-service/ui/src/components/Alerts.tsx:131]: Got it, thanks!

**Pull request merged**